examples/mne/p300_mne_test.py

```diff
@@ -44,7 +47,7 @@
     training=True,
     max_num_options=10,
     max_decisions=50,
-    pp_low=0.1,
+    pp_low=1,
```

Comment:
Why change this, was it giving errors?

Reply:
In the bci_data.py main() docs, pp_low is defined as an int. It seemed to be implemented as a float in all of the other files, so I just changed it from 0.1 to 1. If you think it's better that it stay as a float I can undo that change and update the docs, just let me know :)

Reply:
Never mind, just saw your next comment. I'll resolve this conversation and update the docs (and undo the changes to pp_low=1)

Reply:
Thanks, I meant to come back and delete this after my other comment, but you beat me to it. Thanks a bunch, Alex!